server/services/departureService.ts

```diff
@@ -29,4 +34,18 @@ export default class DepartureService {
 
     return { ...departure, dateTime: parseISO(departure.dateTime).toLocaleDateString('en-GB') }
   }
+
+  async getReferenceData(): Promise<DepartureReferenceData> {
+    const referenceDataClient = this.referenceDataClientFactory(this.token)
+
+    return Promise.all([
```

Comment:
This would be easier to read if you used `await`

Reply:
I'm not sure I follow - sorry!

Reply:
So literally just?:

```
return {
      departureReasons: await referenceDataClient.getReferenceData('departure-reasons'),
      moveOnCategories: await referenceDataClient.getReferenceData('move-on-categories'),
      destinationProviders: await referenceDataClient.getReferenceData('destination-providers')
}
```

I had this before, but @andrewrlee suggestes using `Promise.all`. Can put it back if you think it's more readable

Reply:
Individual preferences 😁 I find await more readable.